Parsers/ZPA/ZPAEvent.txt

```diff
@@ -0,0 +1,292 @@
+            HttpUserAgentOriginal=column_ifexists('UserAgent', ''),
+            HttpRequestXff=column_ifexists('XFF', ''),
+            NameID=column_ifexists('NameID', ''),
+            HttpStatusCode=column_ifexists('StatusCode', ''),
+            HttpRequestBodyBytes=column_ifexists('RequestSize', ''),
+            HttpResponseBodyBytes=column_ifexists('ResponseSize', ''),
+            SrcPortNumber=column_ifexists('ClientPublicPort', ''),
+            EventResultDetails=column_ifexists('ConnectionReason', ''),
+            CorsToken=column_ifexists('CorsToken', ''),
+            Origin=column_ifexists('Origin', '')
+       | project
+            LogTimestamp,
+            Customer,
+            SessionID,
+            ConnectionID,
+            EventResult,
+            DvcAction,
+            NetworkProtocol,
+            DoubleEncryption,
+            DstUserName,
+            DstPortNumber,
+            SrcIpAddr,
+            SrcNatIpAddr,
+            SrcGeoLatitude,
+            SrcGeoLongitude,
+            SrcGeoCountry,
+            ClientZEN,
+            NetworkRuleName,
+            Connector,
+            ConnectorZEN,
+            ConnectorIP,
+            ConnectorPort,
+            Host,
+            Application,
+            AppGroup,
+            DstDomainHostname,
+            DstIpAddr,
+            PolicyProcessingTime,
+            CAProcessingTime,
+            ConnectorZENSetupTime,
+            ConnectionSetupTime,
+            ServerSetupTime,
+            AppLearnTime,
+            TimestampConnectionStart,
+            TimestampConnectionEnd,
+            TimestampCATx,
+            TimestampCARx,
+            TimestampAppLearnStart,
+            TimestampZENFirstRxClient,
+            TimestampZENFirstTxClient,
+            TimestampZENLastRxClient,
+            TimestampZENLastTxClient,
+            TimestampConnectorZENSetupComplete,
+            TimestampZENFirstRxConnector,
+            TimestampZENFirstTxConnector,
+            TimestampZENLastRxConnector,
+            TimestampZENLastTxConnector,
+            SrcBytes,
+            ZENBytesRxClient,
+            DstBytes,
+            ZENBytesTxClient,
+            ZENTotalBytesRxConnector,
+            ZENBytesRxConnector,
+            ZENTotalBytesTxConnector,
+            ZENBytesTxConnector,
+            Idp,
+            Version,
+            ZEN,
+            CertificateCN,
+            PrivateIP,
+            DvcIpAddr,
+            Latitude,
+            Longitude,
+            CountryCode,
+            TimestampAuthentication,
+            TimestampUnAuthentication,
+            DvcHostname,
+            DvcOs,
+            DvcType,
+            TrustedNetworks,
+            TrustedNetworksNames,
+            SAMLAttributes,
+            PosturesHit,
+            PosturesMiss,
+            ZENLatitude,
+            ZENLongitude,
+            ZENCountryCode,
+            SessionType,
+            DvcOS,
```

Comment:
Does this should be "SrcDvcOs"? Please refer the https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema

Reply:
updated